angular/lib/src/core/linker/app_view.dart

```diff
@@ -584,7 +583,7 @@ abstract class AppView<T> {
     domRootRendererIsDirty = true;
   }
 
-  VoidFunc1<E> eventHandler0<E>(VoidFunc0 handler) {
+  void Function(E) eventHandler0<E>(void Function() handler) {
```

Comment:
I think we decided always the former, the latter may be removed in a future version of Dart.